apps/studio/components/interfaces/Settings/API/PostgrestConfig.tsx

```diff
@@ -98,163 +129,213 @@ const PostgrestConfig = () => {
+                                  still exposed over our GraphQL endpoints.
+                                </p>
+                                <Button asChild type="default">
+                                  <Link
+                                    href={`/project/${projectRef}/database/extensions`}
+                                    // className="!no-underline !hover:bg-surface-100 !text-foreground"
+                                  >
+                                    Disable the pg_graphql extension
+                                  </Link>
+                                </Button>
+                              </>
                             )}
-                          </div>
-                        )}
-                      </FormSectionContent>
-                    )}
-                  </FormSection>
+                          </>
+                        </Admonition>
+                      )}
+                    </FormItemLayout>
+                  </FormItem_Shadcn_>
+                )}
+              />
+
+              <FormField_Shadcn_
+                control={form.control}
+                name="dbExtraSearchPath"
+                render={({ field }) => (
+                  <FormItem_Shadcn_ className="w-full">
+                    <FormItemLayout
+                      className="w-full px-8 py-8"
+                      layout="horizontal"
+                      label="Extra search path"
+                      description="Extra schemas to add to the search path of every request. Multiple schemas must be comma-separated."
+                    >
+                      <FormControl_Shadcn_>
+                        <Input_Shadcn_
+                          size="small"
+                          disabled={!canUpdatePostgrestConfig}
+                          {...field}
+                        />
+                      </FormControl_Shadcn_>
+                    </FormItemLayout>
+                  </FormItem_Shadcn_>
+                )}
+              />
+
+              <FormField_Shadcn_
+                control={form.control}
+                name="maxRows"
+                render={({ field }) => (
+                  <FormItem_Shadcn_ className="w-full">
+                    <FormItemLayout
+                      className="w-full px-8 py-8"
+                      layout="horizontal"
+                      label="Max rows"
+                      description="The maximum number of rows returned from a view, table, or stored procedure. Limits payload size for accidental or malicious requests."
+                    >
+                      <FormControl_Shadcn_>
+                        <Input_Shadcn_
+                          size="small"
+                          disabled={!canUpdatePostgrestConfig}
+                          {...field}
+                          type="number"
+                          {...form.register('maxRows', {
```

Comment:
wondering why this one has valueAsNumber: true and the one below doesn't. 
just going to check